api/src/main/java/org/openmrs/module/registrationapp/converter/ExtensionConfigGenerator.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.registrationapp.converter;
+
+import org.codehaus.jackson.JsonNode;
+import org.openmrs.module.appframework.domain.Extension;
+import org.openmrs.module.appframework.domain.AppDescriptor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+
+public class ExtensionConfigGenerator {
```

Comment:
If this class is only intended to generate one config, then its name must reflect this. So this guy should be named something like `RegistrationSummaryExtensionsGenerator`.

Also always generate JavaDoc for new classes and methods.

Reply:
Corrected.